From 87e45cbcd06b82783e4165c52148b7a166009c88 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Tue, 16 Nov 2021 12:21:51 -0800
Subject: [PATCH] Update for python 3.10

---
 m4/ax_python_devel.m4 | 28 ++++++++++++++--------------
 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git a/m4/ax_python_devel.m4 b/m4/ax_python_devel.m4
index e0804c5..e562ca3 100644
--- a/m4/ax_python_devel.m4
+++ b/m4/ax_python_devel.m4
@@ -138,20 +138,20 @@ variable to configure. See ``configure --help'' for reference.
 		fi
 	fi
 
-	#
-	# Check if you have distutils, else fail
-	#
-	AC_MSG_CHECKING([for the distutils Python package])
-	ac_distutils_result=`$PYTHON -c "import distutils" 2>&1`
-	if test -z "$ac_distutils_result"; then
-		AC_MSG_RESULT([yes])
-	else
-		AC_MSG_RESULT([no])
-		AC_MSG_ERROR([cannot import Python module "distutils".
-Please check your Python installation. The error was:
-$ac_distutils_result])
-		PYTHON_VERSION=""
-	fi
+# 	#
+# 	# Check if you have distutils, else fail
+# 	#
+# 	AC_MSG_CHECKING([for the distutils Python package])
+# 	ac_distutils_result=`$PYTHON -c "import distutils" 2>&1`
+# 	if test $? -eq 0; then
+# 		AC_MSG_RESULT([yes])
+# 	else
+# 		AC_MSG_RESULT([no])
+# 		AC_MSG_ERROR([cannot import Python module "distutils".
+# Please check your Python installation. The error was:
+# $ac_distutils_result])
+# 		PYTHON_VERSION=""
+# 	fi
 
 	#
 	# Check for Python include path
-- 
2.33.0

